custom/enikshay/user_setup.py

```diff
@@ -267,50 +269,104 @@ def __init__(self, *args, **kwargs):
         super(ENikshayNewMobileWorkerForm, self).__init__(*args, **kwargs)
 
 
-class ENikshayUserDataEditor(CustomDataEditor):
-    """Custom User Data (everywhere it turns up)"""
-    def __init__(self, *args, **kwargs):
-        super(ENikshayUserDataEditor, self).__init__(*args, **kwargs)
+# pcp -> MBBS
+# pac -> AYUSH/other
+# plc -> Private Lab
+# pcc -> pharmacy / chemist
+# dto -> Field officer??
```

Comment:
that column in the spec was detailing the location type that should be created when someone selects that particular user type. For the field officer, I don't think they get a new location created, but instead are assigned to the DTO level (AFAIK, I think we were going to check this with Erin, which is why I put it in parens). 

I don't know where in this code that happens, if at all, so I'm assuming this is resolved.